main/src/cgeo/geocaching/connector/gc/GCConstants.java

```diff
@@ -172,8 +172,8 @@
     final static Pattern PATTERN_USERTOKEN = Pattern.compile("userToken\\s*=\\s*'([^']+)'");
     final static Pattern PATTERN_LIST_PQ = Pattern.compile(Pattern.quote("(") + "(\\d+)" + Pattern.quote(")") + ".+?guid=(.+?)\".*?title=\"(.+?)\"");
 
-    // downloadable PQs,     <a href="/pocket/downloadpq.ashx?g=b1390cb5-88f3-4fcc-b5e9-f3786c30f773&src=web">
-    final static Pattern PATTERN_LIST_PQ_DL = Pattern.compile("downloadpq.ashx\\?g=([a-zA-Z0-9-]+).*?>\\s*(\\d+)\\s*</td>");
+    // downloadable PQs,
+    final static Pattern PATTERN_LIST_PQ_DL = Pattern.compile("downloadpq\\.ashx\\?g=([a-fA-F0-9-]+)[^>]*>\\s*([^<]*)<\\/a>\\s*<\\/td>\\s*<td[^>]*>[^<]*<\\/td>\\s*<td[^>]*>\\s*(\\d+)\\s*<\\/td>[^<]*<td>\\s*([^<(]*)(\\(([^\\s]+)[^<]*)?<");
```

Comment:
Some time ago I helped with the pocket query creator app and there the Jericho HTML Parser is used. That would be more stable than using Regex Patterns.

BTW: do you know this? https://stackoverflow.com/questions/1732348/regex-match-open-tags-except-xhtml-self-contained-tags/1732454#1732454
:)